Makefile

```diff
@@ -5,6 +5,7 @@ BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
 all:
+	ln -sfn $(CURDIR) $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 	go build -tags "$(BUILDTAGS)" -o runc .
```

Comment:
@tianon Re GOPATH, sounds good.
Re .gitignore - not sure I understand why we'd want to do that. If someone changes this symlink, we'd want to be able to track changes, wouldn't we?